docs/site/migration/LB3-vs-LB4-booting.md

```diff
@@ -0,0 +1,148 @@
+---
+lang: en
+title: 'Differences in LoopBack 3 and LoopBack 4 booting process'
+keywords: LoopBack 4.0, LoopBack 4, LoopBack 3.0, LoopBack 3
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/LB3-vs-LB4-booting.html
+---
+
+## Differences between LoopBack 3 and LoopBack 4 booting process
+
+The booting process in LoopBack 3 is handled by
+[loopback-boot](https://github.com/strongloop/loopback-boot),
+which performs the following tasks:
+
+### In LoopBack 3
```

Comment:
I feel this paragraph 
> The booting process in LoopBack 3 is handled by
[loopback-boot](https://github.com/strongloop/loopback-boot),
which performs the following tasks:

should be under ### In LoopBack 3, instead of before it.

Reply:
Indeed.